src/taipy/rest/api/resources/job.py

```diff
@@ -277,8 +276,6 @@ def __init__(self, **kwargs):
     @_middleware
     def post(self, job_id):
         manager = _JobManagerFactory._build_manager()
-        job = manager._get(job_id)
-        if not job:
-            return make_response(jsonify({"message": f"Job {job_id} not found"}), 404)
+        job = _get_or_raise(job_id)
         manager._cancel(job)
         return {"message": f"Cancelled job {job_id}"}
```

Comment:
Check if this should be worded just like the other message: (only one 'l' by the way)

     Job {} was canceled.